packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -394,8 +387,8 @@ void _upgradeMap(Map<String, dynamic> map, ServiceObjectOwner owner) {
 }
 
 void _upgradeList(List<dynamic> list, ServiceObjectOwner owner) {
-  for (int i = 0; i < list.length; i++) {
-    final dynamic v = list[i];
+  for (int i = 0; i < list.length; i += 1) {
```

Comment:
Curious: is there a reason why `i++` is less desirable?

Reply:
in this case, not really. In the general case, `++` suffix can be less efficient (since it has to store the value, then increment it, then use it), so `++` prefix is preferred. But that's ugly. So I pre-emptively turn `++` into the equivalent `+= 1` to make people more comfortable with `+= 1` so that the issue of `++` prefix can go away. :-)